src/main/kotlin/com/exactpro/th2/codec/json/JsonPipelineCodec.kt

```diff
@@ -70,6 +70,10 @@ class JsonPipelineCodec(
     private val messageTypePointer: JsonPointer
 
     init {
+        if (settings.messageTypeDetection == CONSTANT) {
+            check(settings.constantMessageType.isNotEmpty()) { "Constant message type parameter is required in CONSTANT mode" }
```

Comment:
This verification is already done by the settings class. We should check that the message specified in the `constantMessageType` parameters exists in the dictionary